main.tf

```diff
@@ -8,7 +8,7 @@ resource "aws_alb" "alb" {
   subnets            = var.subnets
   load_balancer_type = var.load_balancer_type
   idle_timeout       = var.idle_timeout
-  ip_address_type    = var.enable_ipv6 ? "dualstack" : "ipv4"
+  ip_address_type    = var.enable_ipv6 ? (var.disable_public_ipv4 ? "dualstack" : "dualstack-without-public-ipv4") : "ipv4"
```

Comment:
Am I misreading the logic here, or is the inner ternary operator reversed? It looks like it will do "dualstack-without-public-ipv4" if `disable_public_ipv4` is false (meaning we want IPv4).

Reply:
Sorry, I see the fix now in a later PR.